src/deploy.ts

```diff
@@ -62,7 +62,7 @@ export interface DeployEffects extends ConfigEffects, TtyEffects, AuthEffects {
   output: NodeJS.WritableStream;
   visitFiles: (root: string) => Generator<string>;
   stat: (path: string) => Promise<Stats>;
-  build: ({config, addPublic}: BuildOptions, effects?: BuildEffects) => Promise<void>;
+  build: ({config}: BuildOptions, effects?: BuildEffects) => Promise<void>;
```

Comment:
I removed the `addPublic` flag which was only used by tests to reduce the complexity of building (slightly). There’s still some public-specific logic in `build-test.ts`.